R/point_interval.R

```diff
@@ -136,7 +136,7 @@ globalVariables(c("y", "ymin", "ymax"))
 #'   ggplot(aes(x = x, y = 0)) +
 #'   stat_halfeye(point_interval = mode_hdi, .width = c(.66, .95))
 #'
-#' @importFrom purrr map_dfr map map2 discard map_lgl iwalk
+#' @importFrom purrr map
```

Comment:
do we still need this import or have all instances of `map()` been replaced with `lapply()` now?